test/verify/check-machines

```diff
@@ -1668,6 +1668,17 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
                                                 start_vm=False,
                                                 connection='session'))
 
+        runner.createThenDeleteInInstallation(TestMachines.VmDialog(self,
```

Comment:
Recently as you noticed I split the testCreate into smaller tests - so that we can group them and not have that long test failing in random places.
Now testCreateFileSource (the test where you just added the new subtest) is not supposed to check the installation, just creation with file source.
For testing the install script we do have the testCreateThenInstall test, were I suggest we should add this new subtest.